src/main/resources/templates/users/notifications.ftlh

```diff
@@ -84,12 +85,10 @@
 
                     <ul class="list-group">
                     @notifications.map { case (notification, origin) =>
-                    <li class="list-group-item notification" data-action="${notification.action!"none"}"
-                    data-id="@notification.id">
-                        @origin.map { origin =>
-                            <#import "*/utils/userAvatar.ftlh" as userAvatar>
-                            <@userAvatar.userAvatar userName=origin.name avatarUrl=origin.avatarUrl clazz = "user-avatar-s"></@userAvatar.userAvatar>
-                        }
+                    <li class="list-group-item notification" data-action="${notification.action!"none"}" data-id="@notification.id">
+                        <#list origin as origin>
```

Comment:
yeah, not sure thats what map does, origin as origin makes zero sense however.

Reply:
Yeah, I was just trying to replicate what the old stuff was doing.